src/train_controlnet_flax.py

```diff
@@ -806,7 +807,9 @@ def main():
             "mid_block",
         ]:
             controlnet_params[key] = unet_params[key]
-
+        rng, rng_params = jax.random.split(rng)
+        controlnet_params['controlnet_cond_embedding']['conv_in']['kernel']=jax.nn.initializers.lecun_normal()(rng_params, (3,3,6,16), jnp.float32)
```

Comment:
The changes I made also affected the `controlnet_cond_embedding` but during initialization. I'm not sure if this identical. I'll try to confirm, but can you as well?

Reply:
The JaX/Flax model doesn't use src/diffusers/models/controlnet.py - it's all rewritten in JaX/Flax in src/diffusers/models/controlnet_flax.py
This new solution is suboptimal, but works without making changes to the underlying library.

Reply:
Okay, I'll try to double check the jax code. If we can avoid changing the underlying library, great. If we have to change it, we have to change it